content/_data/changelogs/weekly.yml

```
@@ -8196,6 +8196,85 @@
 
   # pull: 4828 (PR title: Fix JavaScript typo)
 
+- version: "2.245"
+  date: 2020-07-15
+  changes:
+  - type: major rfe
+    category: major rfe
+    pull: 4823
+    references:
+      - pull: 4823
+      - url: https://www.jenkins.io/doc/administration/requirements/windows/
+        title: Windows support policy
+    authors:
+    - NextTurn
+    - oleg-nenashev
+    message: |-
+      Stop supporting .NET Framework 2.0 for launching Jenkins server and agents as a Windows service.
+      .NET Framework 4.0 or above is now required.
+  - type: major rfe
+    category: major rfe
+    pull: 4823
+    references:
+      - pull: 4823
+      - url: https://github.com/winsw/winsw/releases
+        title: Windows Service Wrapper changelog
+      - url: https://github.com/jenkinsci/windows-slave-installer-module/releases/tag/windows-slave-installer-2.0
+        title: Windows Agent Installer 2.0 Changelog
+    authors:
+    - NextTurn
+    - oleg-nenashev
+    message: |-
+      Update Windows Service Wrapper from 2.3.0 to 2.9.0.
+      It includes numerous improvements and bugfixes.
+      Most notably, the service installer will now ask for permission elevation if the required permissions are missing.
+  - type: bug
+    category: bug
+    pull: 4758
+    issue: 62949
+    authors:
+    - Absh-Day
+    message: |-
+      Escape special characters from href attrib.
+  - type: rfe
+    category: internal
+    pull: 4831
+    references:
+      - pull: 4758
+      - issue: 62949
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.4
+        title: Remoting 4.4 changelog
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.5
+        title: Remoting 4.5 changelog
+    authors:
+    - jeffret-b
+    message: |-
+      Internal: Upgrade to Remoting 4.5.
+      This switches agent.jar and remoting.jar to a code-signing certificate owned by the CDF.
+  - type: rfe
```

I did some conservative merging, didn't want to add too much due to security release incoming:

```suggestion
  - type: rfe
    category: "rfe"
    pull: 4841
    issue: 61806
    authors:
    - jglick
    message: |-
      When configuring your **User Defined Time Zone**, you will now see hints about the current time on the server’s zone and, if not selecting **Default**, in the selected zone.
  - type: rfe
    category: "developer"
    pull: 4843
    references:
      - url: https://github.com/jenkinsci/dark-theme
        title: Dark Theme repository
    authors:
    - timja
    message: |-
      Developer: Allow theming text area handle
  - type: rfe
```